.eslintignore

```diff
@@ -2,4 +2,5 @@ artifacts/*
 js/viz/docs/*
 node_modules/*
 testing/helpers/sinon/*
-/themebuilder/data/metadata/*
+themebuilder/data/metadata/*
+themebuilder-scss/**/*
```

Comment:
https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy
>To limit ESLint to a specific project, place "root": true inside the eslintConfig field of the package.json file or in the .eslintrc.* file at your project's root level. ESLint will stop looking in parent folders once it finds a configuration with "root": true.

So, let's add `"root": true ` to [ThemeBuilder config](https://github.com/DevExpress/DevExtreme/pull/13036/files#diff-cf5d90eb805257fe74ea00764ceb6771)?

Reply:
See https://github.com/DevExpress/DevExtreme/pull/13036#discussion_r427819392